frontend/src/pages/participatedMentoring/StarRating/StarRating.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled';
+import filledStar from '../../../common/assets/images/starIcon.svg';
+import emptyStar from '../../../common/assets/images/emptyStarIcon.svg';
+
+interface StarRatingProps {
+  rating: number;
+  onRatingChange: (rating: number) => void;
+}
+
+function StarRating({ rating, onRatingChange }: StarRatingProps) {
+  return (
+    <StyledContainer>
+      {Array.from({ length: 5 }, (_, index) => {
+        const score = index + 1;
+        const isFilled = score <= rating;
+        return (
+          <StyledStarButton
+            key={score}
+            type="button"
+            onClick={() => onRatingChange(score)}
+          >
+            <StyledStarIcon src={isFilled ? filledStar : emptyStar} alt="" />
```

Comment:
p4: 구현하다 바빠서 alt에 값을 빠트린 것 같아! 😊 혹시 `${score}점`이런 건 어떨까?🤔

```suggestion
            <StyledStarIcon src={isFilled ? filledStar : emptyStar} alt={`${score}점`} />
```

Reply:
[refactor: 별점 이미지 alt 태그 추가](https://github.com/woowacourse-teams/2025-Fit-toring/pull/417/commits/864a9751c388774af49575e5af534b1d9d0681c2)
허거덩 수정했어 👍